mtv-plan/mtv.sw.yaml

```diff
@@ -119,5 +121,5 @@ states:
               topic: "MTV plan assessment workflow"
               severity: "normal"
     stateDataFilter:
-      output: '{workflowOptions: {migrationOptions: [{id: "mtv-migration", name: "MTV migration workflow"}]}, assessmentResult: {passed: true, message: "plan is ready for migration"}}'
+      output: '{workflowOptions: {migrationOptions: [{id: "mtv-migration", name: "MTV migration workflow"}]}}'
```

Comment:
this output is not good, it does not matches the schema defined by `mtv-plan/schemas/workflow-output-schema.json`:

https://github.com/parodos-dev/serverless-workflows/pull/439/files#diff-a1e81bb077285ef5ac194f923417a1ae412c439357a9f3d725d909d9a84b2187R106 is good

You should have something similar here:
```
      output: '{result: {completedWith: "success", message: "MTV plan successful", "nextWorkflows": [{id: "mtv-migration", name: "MTV migration workflow"}], "outputs": [] }}'
```

Reply:
fixed